flytekit/tools/lazy_loader.py

```diff
@@ -26,12 +26,16 @@ def get_extras_require(cls):
         """
         d = cls.LAZY_LOADING_PLUGINS.copy()
         all_plugins = []
+        all_plugins3 = []
```

Comment:
@akhurana001 there is another bug, when you install flyte[notebook], you automatically install spark, because the import has spark as a dependency, can you remove that as well?

Reply:
Not sure what you mean. There is no `flytekit[notebook]` currently afaik ?